CHANGES.rst

```diff
@@ -9,6 +9,9 @@
 
 - Add support for Python 3.9.
 
+- Add support for ``zope.untrustedpython`` on Python 3. With it, Python
+  expressions are now protected. It is activated using the ``untrusted`` extra.
+
```

Comment:
I wonder if this is a significant-enough change to bump the upcoming version to 5.0.0?

Reply:
Done in https://github.com/zopefoundation/zope.pagetemplate/pull/29/commits/1a29ae248df38b66f795b4142c325395adb69ca3